draft-ietf-oauth-sd-jwt-vc.md

```diff
@@ -356,19 +358,27 @@ JWT MUST adhere to the rules defined in Section 5.3 of
 The Key Binding JWT MAY include additional claims which, when not understood, MUST
 be ignored by the Verifier.
 
-## Examples
+## Examples {#presentation-examples}
 
 The following is a non-normative example of a presentation of the SD-JWT shown in (#vc-sd-jwt-example) including a Key Binding JWT.
-In this presentation, the Holder provides only the Disclosure for the  `address` claim.
+In this presentation, the Holder provides only the Disclosures for the  `address` and `is_over_65` claims.
```

Comment:
```suggestion
In this presentation, the Holder provides only the Disclosures for the `address` and `is_over_65` claims.
```

Reply:
Pretty sure extra spaces are overcome somewhere in the document building process but fixing it early can't hurt